spec/controllers/token_based_resume_controller_spec.rb

```diff
@@ -56,14 +72,52 @@
     end
 
     it "load correct application given code" do
+      UnaccompaniedMinor.where.not(reference: uam.reference).destroy_all
+
       parms = { abstract_resume_token: { token: sms_code }, uuid: magic_id }
 
       post :submit, params: parms
 
       expect(response.status).to eq(200)
-      puts flash[:error]
       expect(response).to render_template("sponsor-a-child/task_list")
     end
+
+    it "shows application select page if user has more than one" do
+      email = "test@example.com".freeze
+      phone_number = "07983111111".freeze
+
+      uam = UnaccompaniedMinor.new(
+        given_name: "neilone",
+        minor_given_name: "Minorone",
+        adult_given_name: "adultgiven",
+        email:,
+        phone_number:,
+      )
+      uam.save!
+
+      uam2 = UnaccompaniedMinor.new(
+        given_name: "neiltwo",
```

Comment:
I think these can be deleted (in both UAMS) - simply there for experiments?

Reply:
nah, let's keep them, they're not harmful